### PR TITLE
TECH-899 Use Jib for ARM images

### PR DESCRIPTION
**Pull request by tonisojandu-sympower**

### CODE REVIEW

File: .github/workflows/release-for-testing.yml [hunk 1]
```diff
@@ -74,24 +73,16 @@ jobs:
...
-          ./gradlew bootBuildImage --parallel --continue --stacktrace \
+          ./gradlew jib --parallel --continue --stacktrace \
+            -Djib.from.platforms=linux/amd64,linux/arm64 \
             -PsympowerMavenRepoPublicUrl=${{ secrets.MAVEN_REPO_PUBLIC_URL }} \
             -PsympowerMavenRepoSnapshotsUrl=${{ secrets.MAVEN_REPO_SNAPSHOTS_URL }} \
             -PsympowerMavenRepoReleasesUrl=${{ secrets.MAVEN_REPO_RELEASES_URL }} \
             -PsympowerMavenRepoUsername=${{ secrets.MAVEN_REPO_USER }} \
             -PsympowerMavenRepoPassword=${{ secrets.MAVEN_REPO_PASS }} \
-            -PregistryHost=${{ secrets.DOCKER_REGISTRY_HOST }} \
+            -PregistryHost="${{ secrets.DOCKER_REGISTRY_HOST }}" \
```
Comment (florianbuth-sympower) [inline .github/workflows/release-for-testing.yml:83]: Is this change intentional?

Comment (tonisojandu-sympower) [reply .github/workflows/release-for-testing.yml:83]: This is good practice with bash to ensure special characters like spaces and other such in the argument would not break argument for the parser. However, in this case I accidentally added the registryHost twice, noticed it later and deleted the old one.

Comment (florianbuth-sympower) [reply .github/workflows/release-for-testing.yml:83]: Ah nice! Should we add it to the other ones as well then, to have it unified?

Comment (tonisojandu-sympower) [reply .github/workflows/release-for-testing.yml:83]: Makes sense, I pushed changes now.
